x/manifest/depinject.go

```diff
@@ -22,10 +22,10 @@ import (
 var _ appmodule.AppModule = AppModule{}
 
 // IsOnePerModuleType implements the depinject.OnePerModuleType interface.
-func (am AppModule) IsOnePerModuleType() {}
+func (a AppModule) IsOnePerModuleType() {}
```

Comment:
convention is am but this is minor (SDK does not follow go best practice)

Reply:
[32d5026](https://github.com/liftedinit/manifest-ledger/pull/69/commits/32d5026fd1cd96fe569b13b7cfa187ece0dc57fb)